components/badge.css

```diff
@@ -0,0 +1,56 @@
+.badges {
+    margin-left: 1rem;
+}
+
+.badge-user-active {
+    font-size: 10px;
+    background-color: var(--green);
+    border: 1.5px solid var(--dark-gray);
+    padding: 1px 6px;
+    border-radius: 50%;
+    position: relative;
+    left: -1.2rem;
+    top: -0.3rem;
+}
```

Comment:
It's a good idea to follow a single sizing unit convention inside a class. Using `rem` makes things responsive a bit.